src/components/organisms/Slider/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import Button from '../../atoms/Button';
+import CarouselText from '../../molecules/CarouselText';
+import './style.scss';
+
+const Slider = ({ title, subtitle, images }) => {
+  const slides = images ? [
+    <img key="1" src={require(`../../../${images[2].src}`).default} alt={images[2].alt} />,
```

Comment:
ok, i will say like that, please use loop to render all slides that you get from structure.json, becouse this file rteason is to mockup sererver response, you solution for carrousel restrict slide count to 3
so basicaly use loops

Reply:
Added map function to render all slides  from json file.